src/CopyrighFixer/CopyrighFixer/signer.cpp

```diff
@@ -5,27 +5,135 @@
 //# of this license document, but changing it is not allowed.
 //#
 
+#include <time.h>
 #include "signer.h"
 #include "CopyrighFixer/ifilemanager.h"
+#include <QDir>
+#include <quasarapp.h>
 
 namespace CopyrighFixer {
 Signer::Signer() {
 
 }
 
+
+bool Signer::processSign(const QString &pathToFile, const Config &objConf) const {
+
+    QDir currentFolder(pathToFile);
+    currentFolder.setFilter(QDir::Dirs | QDir::Files | QDir::NoDotAndDotDot);
+
+    QFileInfoList folderItems(currentFolder.entryInfoList());
+
+    for (auto i_file: folderItems) {
+
+        if (i_file.isDir()) {
+            processSign(i_file.filePath(), objConf);
```

Comment:
i think you forget invoke this **continue** operator but i am recommended use in this case: if else operator. i think it will more readable